### PR TITLE
remove api.test_architecture.test_positive_post_hash because it's no longer supported

### PR DESCRIPTION
**Pull request by rplevka**
addresses https://github.com/SatelliteQE/robottelo/issues/4042

### CODE REVIEW

**Comment by oshtaier** [comment]
ACK pending removing unused imports

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/api/test_architecture.py [hunk 1]
```diff
@@ -27,35 +25,6 @@ class ArchitectureTestCase(APITestCase):
     """Tests for architectures."""
 
     @tier2
-    def test_positive_post_hash(self):
-        """Do not wrap API calls in an extra hash.
-
-        @id: 44654ec5-5211-4326-bcad-9824f36a036f
-
-        @Assert: It is possible to associate an activation key with an
```
Comment (sghai) [inline tests/foreman/api/test_architecture.py]: wrong assert statement. typo here as we are associating arch but activation key pointed. Anyways the whole test no longer required.

**Comment by sghai** [comment]
ACK

**Pull request merged**